deployment/arm/deploy-cosmosDB.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I like to add flag to bash to exist early "set -e"

Reply:
As per the discussion with Wallace, deleting this script and will deploy the Cosmos as part of the global deploymentt